src/renderer/src/helpers/StringHelper.ts

```diff
@@ -6,6 +6,14 @@ export class StringHelper {
     return str
   }
 
+  static truncateStringStart(str: string, maxLength: number) {
```

Comment:
What do you think? 

```typescript
static truncateTextStart(text: string, maxLength: number) { ... }
```

Reply:
I followed the pattern of the other methods in the class